src/main/java/core/strategy/impl/PurchaseOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.strategy.impl;
+
+import core.model.Fruit;
+import core.storage.DataBase;
+import core.strategy.FruitStrategy;
+
+public class PurchaseOperation implements FruitStrategy {
```

Comment:
I would suggest creating separate interface Operation or OperationHandler with execute() or some other method and implement that one for all operations